lib/sum_of_multiples.rb

```diff
@@ -0,0 +1,22 @@
+class SumOfMultiples
+  @number
+  @another_number
+  def initialize(a, b)
```

Comment:
Parameters name should be proper. Look how to names parameters for your constructor.